tlddoc/pom.xml

```diff
@@ -0,0 +1,47 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>tlddoc</artifactId>
+    <name>taglibrarydoc</name>
+    <groupId>taglibrarydoc</groupId>
+    <version>2.0-SNAPSHOT</version>
+    <url>http://taglibrarydoc.dev.java.net</url>
+    <licenses>
+        <license>
+            <name>BSD License</name>
+            <url>
+http://taglibrarydoc.dev.java.net/source/browse/*checkout*/taglibrarydoc/taglibrarydoc/license.txt
+            </url>
```

Comment:
Does this link actually work for you?

Reply:
Didn't think too much about the URL.... Looks like a bad link.  Should I remove the URL part entirely? Or keep it as it is?

Closest item I can find so far is : http://web.archive.org/web/20070808225932/http://www.opensource.org/osi3.0/licenses/bsd-license.php  
- Linked from http://web.archive.org/web/20070624180825/https://taglibrarydoc.dev.java.net/

Reply:
http://web.archive.org/web/20070624180825/https://taglibrarydoc.dev.java.net/ points to : http://web.archive.org/web/20070808225932/http://www.opensource.org/osi3.0/licenses/bsd-license.php

So I think we're safe to point to: http://opensource.org/licenses/BSD-3-Clause

Reply:
I agree.  Both list 3 clauses, we should be good.  

I've updated the pom.